javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,20 @@
+var fullName = 'Laura Grace Manimtim';
+var isCool = true;
+var totalPets = 2;
+
+console.log('value of fullName: ' + fullName);
```

Comment:
you want to replace the `+` in your console logs with `,` so that you are not concatenating the value from the variable to the console log label.